include/mgos_pppos.h

```diff
@@ -124,6 +124,11 @@ struct mg_str mgos_pppos_get_imei(int if_instance);
 struct mg_str mgos_pppos_get_imsi(int if_instance);
 struct mg_str mgos_pppos_get_iccid(int if_instance);
 
+//raw
+char* mgos_pppos_get_imei_raw(int if_instance);
```

Comment:
* use `const char *` return type in C, keep `char *` in js (i know, it;s confusing, sorry).
* use `_js` suffix rather than `_raw` (local convention)
* no need to declare in `.h` file